settings.gradle

```diff
@@ -1 +1 @@
-include ':app'
+include ':app', ':chat', ':core'
```

Comment:
Now that you are separating the module based on the functionality - does it make sense to rename the app to `video`?

Reply:
I think it's okay to leave it as app for now as this is a common naming convention for the main application module. However, we could probably move the SDK code related to video to it's own video module, but this would be a separate tech debt effort